.travis_script.sh

```diff
@@ -17,6 +17,7 @@ then
     exit $conf
 fi
 export PYTHONPATH=$(pwd)/tools/pylib/:$PYTHONPATH
+export MAKEFLAGS="-j 2"
 time make || exit
 time make check-unit-tests || exit
```

Comment:
[Looks like](https://www.gnu.org/software/make/manual/html_node/Options_002fRecursion.html) it should do "the right thing"